src/triggered/wizards.ts

```diff
@@ -0,0 +1,195 @@
+import { html } from 'lit-element';
+import { get } from 'lit-translate';
+
+import '../filtered-list.js';
+
+import { List } from '@material/mwc-list';
+import { SingleSelectedEvent } from '@material/mwc-list/mwc-list-foundation';
+
+import {
+  createElement,
+  EditorAction,
+  newWizardEvent,
+  Wizard,
+  WizardAction,
+  WizardInput,
+} from '../foundation.js';
+import { clientIcon, controlBlockIcons, inputIcon } from '../icons.js';
+import {
+  Connection,
+  getConnection,
+  getConnectionIndexOf,
+} from './CommunicationMapping.js';
+
+function sinkPrimary(sink: Element): string {
+  const ln =
+    (sink.getAttribute('prefix') ?? '') +
+    sink.getAttribute('lnClass') +
+    (sink.getAttribute('lnInst') ?? '');
+
+  return sink.tagName === 'ClientLN'
+    ? ln
+    : ln +
+        '>' +
+        sink.getAttribute('doName') +
+        '.' +
+        (sink.getAttribute('daName') ?? '');
+}
+
+function sinkSecondary(sink: Element): string {
+  return sink.tagName === 'ClientLN' ? '' : sink.getAttribute('ldInst') ?? '';
+}
+
+function disconnectExtRef(extRef: Element): EditorAction {
+  if (extRef.getAttribute('intAddr')) {
+    // cannot delete extRef but only specific attribute
+    const newExtRef = createElement(extRef.ownerDocument, 'ExtRef', {});
+    if (extRef.getAttribute('intAddr'))
```

Comment:
Codacy has a fix for the issue: [Expected { after 'if' condition.](https://app.codacy.com/gh/openscd/open-scd/pullRequest?prid=7203429)
```suggestion
    if (extRef.getAttribute('intAddr'))
```